pkg/controllers/nodeclaim/termination/controller.go

```diff
@@ -104,6 +107,9 @@ func (c *Controller) Finalize(ctx context.Context, nodeClaim *v1beta1.NodeClaim)
 			}
 			return reconcile.Result{}, client.IgnoreNotFound(fmt.Errorf("removing termination finalizer, %w", err))
 		}
+		NodeClaimTerminationDuration.With(map[string]string{
+			metrics.NodePoolLabel: nodeClaim.Labels[v1beta1.NodePoolLabelKey],
+		}).Observe(time.Since(terminationStatus.LastTransitionTime.Inner.Time).Seconds())
```

Comment:
```suggestion
		}).Observe(time.Since(nodeClaim.StatusConditions().GetCondition(v1beta1.Terminating).LastTransitionTime.Inner.Time).Seconds())
```

nit: consider inlining